hrcase/CRM/HRCase/Upgrader.php

```diff
@@ -116,14 +116,26 @@ public function upgrade_1402() {
     $this->up1402_removeUnusedCaseTypes();
     //Removes CiviCase activity types which should belong to CiviTask component
     $this->removeActivityTypesList($defaultActivityTypes, 'CiviCase');
-    $this->up1402_createDefaultCaseTypes($defaultCaseTypes);
+    $this->createOrUpdateDefaultCaseTypes($defaultCaseTypes);
     $this->up1402_createDefaultActivityTypes($defaultActivityTypes);
     $this->changeActivityTypeComponent('Open Case', 'CiviCase', 'CiviTask');
 
     return TRUE;
   }
 
   /**
+   * Resets all default case types discarding any customization to match new
+   * activity workflow
+   */
+  public function upgrade_1404() {
+    $this->installActivityTypes('CiviDocument', ['P45']);
```

Comment:
Yes, which led to a bit of confusion for me about which repo to do this in since the other types are created in the [T&A repo](https://github.com/compucorp/civihr-tasks-assignments/blob/master/uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php#L501)